hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/ingest/IngestChecker.java

```diff
@@ -274,7 +274,7 @@ private void assertThrottlingPreconditions(
             // at consensus, and adding them to normal throttle buckets, c.f.
```

Comment:
Comment should be modified to use `NOT_SUPPORTED`

Reply:
Good catch! Thanks! Fixed